src/connectors/infinite-hits/__tests__/connectInfiniteHits-test.ts

```diff
@@ -978,4 +978,177 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/infinite-hi
       expect(actual.page).toEqual(2);
     });
   });
+
+  describe('sendEvent & bindEvent', () => {
```

Comment:
maybe call this `insights` in all connectors? It's slightly inconsistent now

Reply:
good call. done https://github.com/algolia/instantsearch.js/pull/4464/commits/d228af5a6ef1b258712c7a4762babd648bbc56b8